ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot

```diff
@@ -65,6 +66,32 @@ Verify User Can Serve And Query A Model
     ...    json_body=${body}    json_header=${header}
     ...    insecure=${TRUE}
 
+Verify Model Upgrade Using Canaray Rollout
+    [Tags]    ODS-XXX    WatsonX
+    [Setup]    Set Project And Runtime    namespace=canary-model-upgrade
+    Create Secret For S3-Like Buckets    endpoint=s3.us-east-2.amazonaws.com/
+    ...    region=us-east-2   namespace=canary-model-upgrade
+    ${flan_isvc_name}=    Set Variable    flan-t5-small-caikit
+    ${model_name}=    Set Variable    flan-t5-small-caikit
+    Compile And Query LLM model   isvc_name=${flan_isvc_name}
+    ...    sa_name=${DEFAULT_BUCKET_SA_NAME}
+    ...    model_storage_uri=s3://ods-ci-wisdom/flan-t5-small/
+    ...    model_name=${model_name}
+    ...    namespace=canary-model-upgrade
+    Log To Console    Applying Canary Tarffic for Model Upgrade
+    Compile And Query LLM Model   isvc_name=${flan_isvc_name}
+    ...    sa_name=${DEFAULT_BUCKET_SA_NAME}
+    ...    model_storage_uri=s3://ods-ci-wisdom/bloom-560m/
+    ...    model_name=${model_name}
+    ...    canaryTrafficPercent=20
+    ...    namespace=canary-model-upgrade
```

Comment:
probably, we could query multiple times and check the answer arrived 20% of the time from the new model and 80% from the previous one

Reply:
We can or may be i will do it in future prs.  howver..the current problem i am facing is that bloom model is not getting downloaded.